ooiservices/app/uframe/controller.py

```diff
@@ -310,3 +310,104 @@ def get_data(stream, instrument,field):
 
     #return jsonify(**resp_data)
     return resp_data
+
+@api.route('/get_profiles/<string:reference_designator>/<string:stream_name>')
+def get_profiles(reference_designator, stream_name):
+
+    data = get_data(reference_designator, stream_name)
+    data = data.json() 
+    # Note: assumes data has depth and time is ordinal
+    # Need to add assertions and try and exceptions to check data
+
+    time = []
+    depth = []
+    preferred_timestamp = data[0]['preferred_timestamp']
+        #preferred_timestamp = preferred_timestamp * 2208988800
+
+    
```

Comment:
Somewhere we need logic that says "If there is no "pressure" field, exit this function.